src/app/component/ProductActions/ProductActions.component.js

```diff
@@ -278,22 +284,66 @@ export default class ProductActions extends PureComponent {
         );
     }
 
-    renderPrice() {
-        const { product: { price, variants, type_id }, configurableVariantIndex } = this.props;
+    renderSchema(name, stockStatus) {
+        const { getLink } = this.props;
+        return (
+            <>
+                <meta itemProp="offerCount" content={ 1 } />
+                <meta itemProp="availability" content={ this.renderStock(stockStatus) } />
+                <a
+                  block="ProductActions"
+                  elem="Schema-Url"
+                  itemProp="url"
+                  href={ getLink() }
+                >
+                    { name }
+                </a>
+            </>
+        );
+    }
 
-        if (type_id === GROUPED) return null;
+    renderPriceWithSchema() {
+        const {
+            product,
+            product: { variants },
+            configurableVariantIndex
+        } = this.props;
 
         // Product in props is updated before ConfigurableVariantIndex in props, when page is opened by clicking CartItem
         // As a result, we have new product, but old configurableVariantIndex, which may be out of range for variants
-        const productOrVariantPrice = variants && variants[configurableVariantIndex] !== undefined
-            ? variants[configurableVariantIndex].price
-            : price;
+        const productOrVariant = variants && variants[configurableVariantIndex] !== undefined
+            ? variants[configurableVariantIndex]
+            : product;
```

Comment:
There is a bug with `productOrVariant` prop, therefore manual work is required. 
When ProductPage generates `productOrVarian` with `configurableVariantIndex === -1`, `productOrVariant` gets `dataSource` value. And unlike `product` (having no selected variant), `dataSource` seems to contain one of the variants. That can be seen by looking at the `{ name }`of both `product` (e.g. Blue pants) and `dataSource` (Blue pants size m).

Reply:
Or is it expected behaviour of `productOrVariant` prop?